src/components/inputs/InlineField.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useCallback, useState } from 'react';
+import { TextInputProps } from 'react-native';
+import Input from './Input';
+import {
+  Column,
+  Columns,
+  Inset,
+  Text,
+  useTextStyle,
+} from '@rainbow-me/design-system';
+
+export type InlineFieldProps = {
+  defaultValue?: string;
+  label: string;
+  placeholder?: string;
+  inputProps?: Partial<TextInputProps>;
+  onChangeText: (text: string) => void;
+  onEndEditing?: TextInputProps['onEndEditing'];
+  validations?: {
+    allowCharacterRegex?: { match: RegExp };
+    maxLength?: { value: number };
+  };
+  value: string;
+};
+
+export default function InlineField({
+  label,
+  onChangeText,
+  placeholder,
+  inputProps,
+  validations,
+  onEndEditing,
+  value,
+}: InlineFieldProps) {
+  const textSize = 16;
```

Comment:
I have a general question about this, I think I saw one related to `height` in other files as well. What's the difference from having this const inside the component and re assigning the const in every re render instead of using it as a const outside the component? Not sure if the `assigning the const in every re render` part is really an issue or not but it's how I was used to do this

Reply:
Yeah, don't think it really matters performance wise either way. But I can hoist it outside